pkg/scheduler/core/generic_scheduler.go

```diff
@@ -107,6 +107,7 @@ type genericScheduler struct {
 	cachedNodeInfoMap        map[string]*schedulercache.NodeInfo
 	volumeBinder             *volumebinder.VolumeBinder
 	pvcLister                corelisters.PersistentVolumeClaimLister
+	pdbLister                algorithm.PDBLister
```

Comment:
Ditto - maybe we can directly use `k8s.io/client-go/listers/policy/v1beta1#PodDisruptionBudgetLister`.

Reply:
Same as above.